src/styles/blocks/button.scss

```
@@ -0,0 +1,26 @@
+.button {
+  box-sizing: border-box;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 60px;
+  border: 1px solid transparent;
+
+  &--first-type {
+    background-color: #cd4d31;
```

don't forget to create variables for all colors
```suggestion
    background-color: $c-color;
```